test/s3path.jl

```diff
@@ -554,6 +557,52 @@ function s3path_tests(config)
             @test !exists(versioned_path)
             @test length(s3_list_versions(config, bucket_name, key)) == 1
         end
+
+        @testset "S3Path null version" begin
+            b = let df = dateformat"yyyymmdd\THHMMSS\Z"
+                "ocaws.jl.test." * lowercase(Dates.format(now(Dates.UTC), df))
+            end
```

Comment:
Would be nice to use a different named bucket for this. Not sure what restrictions are available so we'll start here